### DISCUSSION TITLE
GeoMap Sparklines or Graph mode for layer tooltip

### DISCUSSION THREAD

**Discussion by tedraymond** [grafana/grafana#57029] [Dashboards, Panels & Visualizations]
Any ideas on having visualization in tooltips for layers. Time series graphs, threahold colors, value mapping?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
